CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # CHANGELOG
 
+## v1.9.0
+
+*May 30th, 2024*
+
+This v1.9.0 release introduces new features and improvements to Hermes.
+
+**Major Features**:
+
+1. Channel Upgrade Compatibility: Hermes now supports channel upgrade events introduced in ibc-go v8, ensuring smoother transitions and enhanced functionality.
+
+3. Dynamic Gas Fees Compatibility: Hermes is now compatible with Skip's feemarket for dynamic gas fees, in addition to Osmosis' implementation, providing more flexibility in gas fee management.
+
+Additionally, this release includes various bug fixes enhancing the stability and performance of Hermes. These fixes address issues with channel and connection creation on unsupported ibc-go versions, event extraction, health-check messages, and more.
+
+### BREAKING CHANGES
+
+- [Telemetry & Metrics](telemetry)
+  - Remove the `telemetry` and `rest-server` feature flags, ensuring Hermes is always built with telemetry and REST support.
+    Both servers can still be disabled in the configuration file, by setting `telemetry.enabled = false` and `rest.enabled = false`, respectively.
+    ([\#3878](https://github.com/informalsystems/hermes/pull/3878))
+
+### BUG FIXES
```

Comment:
Let's move this section after the features

Reply:
Done in 37744d55428313f595a042df6b3924948cdc5e8c